### ISSUE TITLE
wiki for annotation should mention gff3

### ISSUE THREAD

**Issue by odoublewen** [igvteam/igv.js#861]
This wiki page does not indicate that `gff3` is an allowed `format` for annotation tracks:
https://github.com/igvteam/igv.js/wiki/Annotation-Track

but clearly it is from looking at the code:
https://github.com/igvteam/igv.js/blob/master/js/feature/featureSource.js#L123

(I'd help out by making a PR to the wiki, but I can't figure out how to do that.)

**Comment by jrobinso** [comment]
gff == gff3, but I'll make it explicit.   "gff2" is actually not supported.

The wiki for a repo is a separate repo,  there's a way to do it but I can't recall the details.

**Comment by odoublewen** [comment]
That's odd.  My gff(3) file does not display any tracks when I have `format='gff'` but works correctly with `format='gff3'`.

Ah, I see, I did not have the (required) header `##gff-version 3` and igv.js must look for that to know what kind of gff it's looking at.

I've fixed my gff files and now it works with `format='gff'`

Thanks!

**Comment by jrobinso** [comment]
The header is technically required, but explicitly setting the format in the config is always a good idea.

**Comment by odoublewen** [comment]
Also, it might be good to mention that only features of type `CDS` are displayed, from what I gather empirically and from https://github.com/igvteam/igv.js/blob/master/js/feature/gffHelper.js

**Comment by jrobinso** [comment]
That's not true.   If you have a specific case where a record is not
displayed open a new issue with a (preferably) small test case.

Transcripts are stitched together from constitutitve parts,  but everything
should be displayed.  GFF is notoriously hard to parse,  BED transcripts
for example are completely non-ambiguous,  but the code there is from
almost 10 years of experience and vetting in IGV.   Still there is always a
new case,  so open a new  issue with a test case if you have a problem.

**Comment by odoublewen** [comment]
OK, I opened a new issue (#870) with a gff3 file with feature types that I expected igv.js to display.
